plans/example/manifest.toml

```diff
@@ -4,6 +4,13 @@ name = "example"
 builder = "exec:go"
 runner = "local:exec"
 
+[builders."docker:generic"]
+  enabled = true
+  [builders."docker:generic".build_args]
+  build_image = "golang:alpine"
+  run_image = "alpine"
```

Comment:
These `build_args` aren't actually used in the Dockerfile.

Can you add a few `RUN echo $ARG1` instructions in the Dockerfile and set them here?

Reply:
I forgot to add these in the actual Dockerfile that got included in this PR but added now so you can see they are used